src/info_page.cpp

```diff
@@ -249,6 +235,23 @@ void InfoPage::refreshInfoWidget()
 	else
 		setStatusLabel(ui->lblIdentifyStatus, tr("Your hardware revision does not support the identify feature"));
 
+	if(m_phoneHome->getDone())
+	{
+		const int checked = dynamic_cast<M2kInfoPage*>(this)->checkLatestFwVersion(m_info_params.value("Firmware version"));
+		if (checked == 1) {
+			ui->lblFirmware->setText("Firmware is up to date!");
```

Comment:
Should this text be wrapped in a `tr()` for translations?

Reply:
fixed